packages/syft/src/syft/types/syft_object.py

```diff
@@ -4,16 +4,19 @@
 import inspect
 from inspect import Signature
 import types
+from typing import AbstractSet
```

Comment:
```suggestion
from collections.abc import Set
```

AbstractSet is [deprecated](https://docs.python.org/3/library/typing.html#typing.AbstractSet) since Python 3.9